server/db/models/rating.js

```diff
@@ -0,0 +1,19 @@
+const Sequelize = require('sequelize')
+const db = require('../db')
+
+const Rating = db.define('rating', {
+  raterId: {
+    type: Sequelize.INTEGER,
+    allowNull: false
+  },
+  personRatingId: {
+    type: Sequelize.INTEGER,
+    allowNull: false
+  },
```

Comment:
do we want the raterId and and personRatingId to be foreign keys, so that we have actual user relations instead of just hard coding numbers?